doc/Caliptra.md

```diff
@@ -233,116 +233,116 @@ In accordance with OCP Attestation specification [Reference 8](#ref-8), devices
 
 Operational security during the manufacturing process is critical, to ensure the DICE entropy is securely initialized, certified, and registered. Operational security avoids any pilfering of this asset by eavesdroppers. Operational security is outside the scope of this specification.
 
-# Threat Model
+# Threat model
 
-This section describes the Caliptra RoT threat model in terms of the profile of the attacks and the attackers that the Caliptra RoT is expected to defend against.
+This section describes the profile of attacks and attackers that Caliptra is expected to defend against.
 
-Threat model as described here takes into account attacker profile, assets and attack surfaces or paths to these assets based on attacker profiles. Subsequent subsections delve into each of these topics.
+The Caliptra threat model describes attacker profiles, assets and attack surfaces, and paths to these assets based on attacker profiles. Subsections provide further details.
 
 Threat scenarios as comprehended by assets and possible attack paths are as complete as possible but focus on the worst case scenarios. Thus not every attack path to asset is captured in this threat model.
 
-## Attacker Profiles
+## Attacker profiles
 
-Attacker profile is the outcome of following factors like tools accessible to the attacker, level of access to the target of evaluation and expertise of the attacker to use these methods. Next level of details of these capabilities scoped for this discussion are as follows.
+Attacker profile is the outcome of factors like the tools that are accessible to the attacker, the level of access to the target of evaluation, and expertise of the attacker to use these methods. These factors are described in the following tables.
 
-*Table 1: Tools Accessible to Attacker*
+*Table 1: Tools accessible to attacker*
 
-| **Attack Tools** | **Type of Attack** | **Purpose/Usage** |
+| **Attack tools** | **Type of attack** | **Purpose and usage** |
```

Comment:
I know it was already here, but "**" bold markers are not needed for table headings, they are already bolded

Reply:
great, thanks; I will pick them up on another pass if there is time.